pkg/splunk/enterprise/indexercluster.go

```diff
@@ -414,6 +424,40 @@ func (mgr *indexerClusterPodManager) getClusterMasterClient() *splclient.SplunkC
 	return mgr.newSplunkClient(fmt.Sprintf("https://%s:8089", fqdnName), "admin", adminPwd)
 }
 
+// getSiteRepFactorOriginCount gets the origin count of the site_replication_factor
+func getSiteRepFactorOriginCount(siteRepFactor string) int32 {
+	re := regexp.MustCompile(".*origin:(?P<rf>.*),.*")
+	match := re.FindStringSubmatch(siteRepFactor)
+	siteRF, _ := strconv.Atoi(match[1])
+	return int32(siteRF)
+}
+
+// verifyRFPeers verifies the number of peers specified in the replicas section
+// of IndexerClsuster CR. If it is less than RF, than we se it to RF.
+func (mgr *indexerClusterPodManager) verifyRFPeers(c splcommon.ControllerClient) error {
+	if mgr.c == nil {
+		mgr.c = c
+	}
+	cm := mgr.getClusterMasterClient()
```

Comment:
In the future it is better to make the **getClusterMasterClient()** and **GetClusterInfo()** APIs independent of the Pod Manager as they are used in MC implementation as well. Mentioned this to @kashok-splunk as a part of her review as well.

Reply:
Yup, probably we can take it as part of another story for future as part of some code clean-up.